src/main/java/com/puppycrawl/tools/checkstyle/checks/naming/AbbreviationAsWordInNameCheck.java

```diff
@@ -213,6 +213,15 @@
  *   String firstURL; // OK, URL abbreviation is allowed
  *   final int TOTAL = 5; // OK, final is ignored
  *   static final int LIMIT = 10; // OK, static final is ignored
+ *   void newOAuth2Client() {} // OK, O abbreviation is allowed
+ *   void OAuth2() {} // OK, O abbreviation is allowed
+ *   void OAUth2() {} // violation, OA abbreviation is not allowed
+ *                    // split occurs as 'OA', 'Uth2'
```

Comment:
Please make this comment the last.
Remove all comments below, they are too internal details.

Reply:
ok, I thought of explaining in detail.

Reply:
done